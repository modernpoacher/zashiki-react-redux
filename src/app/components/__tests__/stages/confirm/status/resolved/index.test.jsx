import React from 'react'
import renderer from 'react-test-renderer'

import Resolved from '#zashiki-react-redux/app/components/stages/confirm/status/resolved'

const MOCK_DEFINITION = {
  meta: {
    uri: '#/'
  },
  elements: {
    title: 'MOCK TITLE',
    description: 'MOCK DESCRIPTION',
    field: {
      id: 'MOCK ID'
    }
  }
}

const MOCK_RESPONSE = {
  '#/': 'MOCK VALUE'
}

describe('#zashiki-react-redux/app/components/stages/confirm/status/resolved', () => {
  describe('Always', () => {
    it('renders', () => {
      const component = (
        <Resolved
          definition={MOCK_DEFINITION}
          response={MOCK_RESPONSE}
          onChange={jest.fn()}
          onSubmit={jest.fn()}
        />
      )

      expect(renderer.create(component).toJSON())
        .toMatchSnapshot()
    })
  })
})
