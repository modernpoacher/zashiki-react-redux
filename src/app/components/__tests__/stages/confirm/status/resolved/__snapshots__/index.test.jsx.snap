// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`#zashiki-react-redux/app/components/stages/confirm/status/resolved Always renders 1`] = `
<div
  className="confirm resolved"
>
  <h2>
    Confirmation
  </h2>
</div>
`;
